src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -28,6 +28,7 @@ public static partial class PlatformDetection
         public static bool IsNetBSD => RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"));
         public static bool IsiOS => RuntimeInformation.IsOSPlatform(OSPlatform.Create("IOS"));
         public static bool IstvOS => RuntimeInformation.IsOSPlatform(OSPlatform.Create("TVOS"));
+        public static bool IsMacCatalyst => RuntimeInformation.IsOSPlatform(OSPlatform.Create("MACCATALYST"));
```

Comment:
Does this need to be commented too?

Reply:
no, it's in test code and the OSPlatform.Create() string based API still works.